lib/spack/spack/cmd/style.py

```diff
@@ -296,7 +312,10 @@ def run_isort(isort_cmd, file_list, args):
 
 @tool("black")
 def run_black(black_cmd, file_list, args):
-    check_fix_args = () if args.fix else ("--check", "--diff", "--color")
+    # always run with config from running spack prefix
+    black_args = ("--config", os.path.join(spack.paths.prefix, "pyproject.toml"))
+    if not args.fix:
+        black_args += ("--check", "--diff", "--color")
```

Comment:
Should color be dependent on where we are running (e.g., we don't want it in a text-based log on CI where we might just see escape sequences?)

Reply:
On GitHub Actions I actually see colored diffs, not sure how they managed to get that working.

Reply:
I'm not sure all other CI services (or places where you might run spack style) handle color well. At least many years ago when I added color to Singularity logging it looked awful (it was either on Travis or Circle).

Reply:
I changed this to follow Spack's color convention (ask `tty.get_color_when()`)